Setup/install.sh

```diff
@@ -80,7 +80,8 @@ install_dependencies() {
     sudo apt-get install python-pip git libi2c-dev python-serial python-rpi.gpio i2c-tools python-smbus arduino minicom libnss-mdns python-dev -y
     sudo pip install -U RPi.GPIO
     sudo pip install pyusb
-    
+    sudo pip install numpy
```

Comment:
Good question. I don't have a DexterOS next to me. But I think numpy is part of Jessie Lite. Hopefully? it's a fairly common library that many things depend on.